### ISSUE TITLE
Change in historical values

### ISSUE THREAD

**Issue by zQuantz** [OxCGRT/covid-policy-tracker#7]
Hello.

I downloaded the file data/OxCGRT_latest.csv at 9:20AM on March 6th, 2020.

I downloaded the same file at 12:04PM on March 6th, 2020.

I noticed that the historical values for the StringencyIndexForDisplay values have changed.

9:20AM Download
           Date CountryName CountryCode  StringencyIndex  StringencyIndexForDisplay
14231  20200111    Portugal         PRT             0.00                       0.00
14232  20200112    Portugal         PRT             0.00                       0.00
14233  20200113    Portugal         PRT             0.00                       0.00
14234  20200114    Portugal         PRT             0.00                       0.00
14235  20200115    Portugal         PRT             0.00                       0.00
14236  20200116    Portugal         PRT             0.00                       0.00
14237  20200117    Portugal         PRT             0.00                       0.00
14238  20200118    Portugal         PRT             0.00                       0.00
14239  20200119    Portugal         PRT             0.00                       0.00
14240  20200120    Portugal         PRT             0.00                       0.00
14241  20200121    Portugal         PRT             0.00                       0.00
14242  20200122    Portugal         PRT             0.00                       0.00
14243  20200123    Portugal         PRT             0.00                       0.00
14244  20200124    Portugal         PRT             0.00                       0.00
14245  20200125    Portugal         PRT             0.00                       0.00
14246  20200126    Portugal         PRT            11.11                      11.11
14247  20200127    Portugal         PRT            11.11                      11.11
14248  20200128    Portugal         PRT            11.11                      11.11
14249  20200129    Portugal         PRT            11.11                      11.11
14250  20200130    Portugal         PRT            11.11                      11.11

12:07PM Download
           Date CountryName CountryCode  StringencyIndex  StringencyIndexForDisplay
14231  20200111    Portugal         PRT              NaN                        NaN
14232  20200112    Portugal         PRT              NaN                        NaN
14233  20200113    Portugal         PRT              NaN                        NaN
14234  20200114    Portugal         PRT              NaN                        NaN
14235  20200115    Portugal         PRT              NaN                        NaN
14236  20200116    Portugal         PRT              NaN                        NaN
14237  20200117    Portugal         PRT              NaN                        NaN
14238  20200118    Portugal         PRT              NaN                        NaN
14239  20200119    Portugal         PRT              NaN                        NaN
14240  20200120    Portugal         PRT              NaN                        NaN
14241  20200121    Portugal         PRT              NaN                        NaN
14242  20200122    Portugal         PRT              NaN                        NaN
14243  20200123    Portugal         PRT              NaN                        NaN
14244  20200124    Portugal         PRT              NaN                        NaN
14245  20200125    Portugal         PRT              NaN                        NaN
14246  20200126    Portugal         PRT              NaN                        NaN
14247  20200127    Portugal         PRT              NaN                        NaN
14248  20200128    Portugal         PRT              NaN                        NaN
14249  20200129    Portugal         PRT              NaN                        NaN
14250  20200130    Portugal         PRT              NaN                        NaN

I use python's pd.read_csv() to directly download the file. I downloaded the file every 30 minutes thereafter and get the same results as the 12:07PM download.

I wanted to either raise the awareness of the potential corrupted data or to understand that this was an intended change that took place on your side.

Thank you so much!

Z

**Comment by TobyPhillips** [comment]
Hi zQuantz,

Yep, that was intentional.

We changed our data structure at the start of last week. And moved from a 7-part stringency index ("legacy index") to a 9-part stringency index. The main stringency index now relies on two new indicators – C4 and C6 – which we didn't have prior to 28 April. We are still in the process of collecting C4 and C6 data for many countries.

This afternoon we realised that for some countries we were reporting stringency index values on days where were missing quite a few indicators – and this could lead to an erroneously low index. So we have removed the stringency index unless there is data recorded for at least 8 of the 9 index components. Because the early days of Portugal are still missing C4 and C6, it only has 7 out of 9 index components.

C4 and C6 should be updated by the weekend. But until then, I would recommend using the legacy index. Hope that helps, but let me know if it's not clear.

Cheers,
Toby
